kogito-editors-js/packages/boxed-expression-component/src/__tests__/components/Table/Table.test.tsx

```diff
@@ -299,13 +296,14 @@ describe("Table tests", () => {
           />
         ).wrapper
       );
-      fireEvent.change(container.querySelector("table tbody tr td textarea")! as HTMLTextAreaElement, {
-        target: { value: newRowValue },
-      });
-      fireEvent.blur(container.querySelector("table tbody tr td textarea")! as HTMLTextAreaElement);
+
+      const textarea = container.querySelector("table tbody tr td textarea") as HTMLTextAreaElement;
+
+      fireEvent.change(textarea, { target: { value: "new value\t" } });
+      // onblur is triggered by Monaco (mock), and the new value relies on Monaco implementation
 
       expect(mockedOnRowsUpdate).toHaveBeenCalled();
-      expect(mockedOnRowsUpdate).toHaveBeenCalledWith([newRow]);
+      expect(mockedOnRowsUpdate).toHaveBeenCalledWith([row]);
```

Comment:
A question about this test case: should the callback be called with update row value (i.e. `row[columnName] = "new value\t"`)?

Reply:
Fixed! Thanks :-)